lib/kamal/commands/builder/clone.rb

```diff
@@ -1,29 +1,31 @@
 module Kamal::Commands::Builder::Clone
-  extend ActiveSupport::Concern
-
-  included do
-    delegate :clone_directory, :build_directory, to: :"config.builder"
-  end
```

Comment:
This delegation becomes superfluous when we need to escape the values returned by `clone_directory` and `build_directory`, so I just removed it. Inlined `config.builder.clone_directory.shellescape` in the single location it's needed, and factorized `config.builder.build_directory.shellescape` in the `escaped_build_directory` method.